src/views/PeopleSearch/components/SearchFieldList/index.tsx

```diff
@@ -173,8 +173,8 @@ const SearchFieldList = ({ onSearch }: Props) => {
    * Whether the user can search for the current params.
    * This prevents a search with empty params, which freezes the client by trying to render thousands of results
    */
-  const canSearch = useMemo(() => {
-    const { includeStudent, includeFacStaff, includeAlumni, ...criteria } = searchParams;
+  const canSearch = (params: PeopleSearchQuery) => {
+    const { includeStudent, includeFacStaff, includeAlumni, ...criteria } = params;
```

Comment:
why was this changed?

Reply:
because we are calling the search function with the params taken from the URL before the setSearchParams gets to update. So the search function will call canSearch, but this time it should check if it can search with the params of the URL instead of the searchParams that was not updated yet